screen_widgets.go

```diff
@@ -96,10 +96,10 @@ type TileDefRequestStyle struct {
 }
 
 type TileDefStyle struct {
-	Palette     *string `json:"palette,omitempty"`
-	PaletteFlip *string `json:"paletteFlip,omitempty"`
```

Comment:
This should remain `*string` because Datadog's API will accept strings or boolean values. If a user (through the json editor in the UI or via the API), sends `"paletteFlip": "true"`, Datadog will accept and return that value in future calls as a string type, and the UI handles that correctly.

Reply:
@nyanshak thanks for the prompt review. Updated the PR and squashed the commits too. Thoughts?

Reply:
Can you update the integration tests to use the new types for `FillMin` and `FillMax`? Specifically, changing L315-316 in `integration/screen_widgets_test.go` to use `datadog.JsonNumber` rather than `datadog.String`?

Reply:
Done, thanks